lib/modules/plugins.js

```diff
@@ -0,0 +1,26 @@
+import actions from './actions';
+
+export const dispatchInitialShell = async (ctx, dispatch) => {
```

Comment:
Can you remove the `async`, I had some trouble getting `await` on session refreshing to work in @2x. `await`'ing functions that were async and didn't call `await` seemed to be the issue ([Fix in 2x](https://github.com/reddit/reddit-mobile/pull/596/commits/e97ae3e50ef63ea35e03bb2f74426ac248548e29))